src/content/configuration/index.md

```diff
@@ -364,6 +364,13 @@ module.exports = {
   </details>
   // Don't follow/bundle these modules, but request them at runtime from the environment
 
+  [serve](https://github.com/webpack-contrib/webpack-serve#webpack-serve): { //object
```

Comment:
Link should be https://github.com/webpack-contrib/webpack-serve#options

Reply:
didnt think of options section for some reason, thanks